helm/cluster-resources/values.yaml

```diff
@@ -1,7 +1,6 @@
 # TEMPLATE-APP: This is set as a reasonable default, feel free to change.
 
 name: cluster-resources
-namespace: kube-system
```

Comment:
Is this defaulted elsewhere?

Reply:
I don't see this being practical. https://github.com/giantswarm/template-app/pull/29